java-frontend/src/test/files/se/StreamConsumedCheck.java

```diff
@@ -13,7 +13,7 @@ void intStream() {
     IntStream range = IntStream.range(0, 10);
     range.count(); // flow@is {{Pipeline is consumed here.}}
     range.average(); // Noncompliant [[flows=is]]
-    range.filter(i -> true); // Noncompliant [[flows=is]]
+    range.filter(i -> true); // Doesn't raise issue, because exception is thrown on previous line and SE is stopped
```

Comment:
As stated before  : This means that stream consumption check should always be run on the engine as it contributes to the shape of the EG.